public/components/header.jsx

```diff
@@ -0,0 +1,16 @@
+import css from '../css/general.module.css'
```

Comment:
header css should be in a separate css file, each component should have it's own css file